test/automl/test_classification.py

```diff
@@ -408,6 +408,14 @@ def test_sparse_matrix_lr(self):
         print(automl_experiment.best_iteration)
         print(automl_experiment.best_estimator)
 
+    def test_reach(self):
+        search_space = {
+            "params": tune.choice(
+                [{"param": "None"}, {"param": tune.qrandint(10, 100, 10)}]
+            ),
+        }
+        return search_space
```

Comment:
This only creates a search space. It won't trigger the exception.
Please make sure your test triggers the exception by testing it locally.